bpf/lib/encap.h

```diff
@@ -139,65 +139,65 @@ __encap_with_nodeid(struct __sk_buff *skb, __u32 tunnel_endpoint,
 	key.tunnel_id = seclabel;
 	key.remote_ipv4 = node_id;
 
-	cilium_dbg(skb, DBG_ENCAP, node_id, seclabel);
+	cilium_dbg(ctx, DBG_ENCAP, node_id, seclabel);
 
-	ret = skb_set_tunnel_key(skb, &key, sizeof(key), BPF_F_ZERO_CSUM_TX);
+	ret = ctx_set_tunnel_key(ctx, &key, sizeof(key), BPF_F_ZERO_CSUM_TX);
 	if (unlikely(ret < 0))
 		return DROP_WRITE_ERROR;
 
-	send_trace_notify(skb, TRACE_TO_OVERLAY, seclabel, 0, 0, ENCAP_IFINDEX,
+	send_trace_notify(ctx, TRACE_TO_OVERLAY, seclabel, 0, 0, ENCAP_IFINDEX,
 			  0, monitor);
 	return 0;
 }
 
 static inline int __inline__
-__encap_and_redirect_with_nodeid(struct __sk_buff *skb, __u32 tunnel_endpoint,
+__encap_and_redirect_with_nodeid(struct __ctx_buff *ctx, __u32 tunnel_endpoint,
 				 __u32 seclabel, __u32 monitor)
 {
-	int ret = __encap_with_nodeid(skb, tunnel_endpoint, seclabel, monitor);
+	int ret = __encap_with_nodeid(ctx, tunnel_endpoint, seclabel, monitor);
 	if (ret != 0)
 		return ret;
 	return redirect(ENCAP_IFINDEX, 0);
 }
 
-/* encap_and_redirect_with_nodeid returns IPSEC_ENDPOINT after skb meta-data is
- * set when IPSec is enabled. Caller should pass the skb to the stack at this
+/* encap_and_redirect_with_nodeid returns IPSEC_ENDPOINT after ctx meta-data is
+ * set when IPSec is enabled. Caller should pass the ctx to the stack at this
  * point. Otherwise returns TC_ACT_REDIRECT on successful redirect to tunnel
- * device. On error returns TC_ACT_SHOT, DROP_NO_TUNNEL_ENDPOINT or
+ * device. On error returns CTX_ACT_DROP, DROP_NO_TUNNEL_ENDPOINT or
```

Comment:
^ No change for `TC_ACT_REDIRECT`

Reply:
Good point, there are two instances inside comments, will convert them with the other follow-up changes.